collect_app/src/main/res/layout/two_item_image.xml

```diff
@@ -27,7 +27,7 @@
             android:layout_alignTop="@+id/image"
             android:layout_toRightOf="@+id/image"
             android:textAppearance="?android:attr/textAppearanceLarge"
-            android:text="text1"/>
+            android:text="@string/text1"/>
```

Comment:
Is it required to have text here? It's just a placeholder so I'm wondering whether it could be omitted instead.